.eslintrc

```diff
@@ -109,6 +111,10 @@
       "no-use-before-define": [2, "nofunc"],
       "no-var-requires": [0],
       "no-unused-vars": 2
+    },
+    "globals": {
```

Comment:
It's better to use a separate `.eslintrc` for client scripts and use `env: { browser: true }` there.